core-java-modules/core-java/src/main/java/com/baeldung/util/SerializationUtils.java

```diff
@@ -0,0 +1,47 @@
+package com.baeldung.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+public class SerializationUtils {
```

Comment:
Rename MySerializationUtils to avoid confusion with the Apache Commons and Spring Core ones.